_posts/2020-11-05-welcome-to-jekyll-demo-post.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Bioinformatic files: types and formats"
+date:   2020-11-06 12:54:04 -0700
+categories: blogpost
+---
+# Introduction
+In bioinformatics there are many different file types to store information from sequencing and -omics work. The most common file types are generally FASTA and FASTQ. But there are a lot of other semi-common file types as well like SAM, BAM, VCF as well as countless other self made file types for individual purposes.
```

Comment:
Succinct! After reading the post, it looks like all four tools are for genomics and not any other -omics. Maybe mention some context at the top to clarify the scope and typical uses of these four formats?